packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -389,12 +390,114 @@ export enum LambdaEdgeEventType {
   VIEWER_RESPONSE = "viewer-response",
 }
 
+interface ViewerCertificateBaseCertficateOptions {
+  /**
+   * How CloudFront should serve HTTPS requests.
+   *
+   * See the notes on SSLMethod if you wish to use other SSL termination types.
+   *
+   * @default SSLMethod.SNI
+   * @see https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_ViewerCertificate.html
+   */
+  readonly sslMethod?: SSLMethod;
+
+  /**
+   * The minimum version of the SSL protocol that you want CloudFront to use for HTTPS connections.
+   *
+   * CloudFront serves your objects only to browsers or devices that support at
+   * least the SSL version that you specify.
+   *
+   * @default - SSLv3 if sslMethod VIP, TLSv1 if sslMethod SNI
+   */
+  readonly securityPolicy?: SecurityPolicyProtocol;
+}
+
+export interface ViewerCertificateAcmCertficateOptions extends ViewerCertificateBaseCertficateOptions {
+  /**
+   * AWS Certificate Manager (ACM) certificate.
+   *
+   * Your certificate must be located in the us-east-1 (US East (N. Virginia)) region to be accessed by CloudFront
+   */
+  readonly certificate: certificatemanager.ICertificate;
+}
+
+export interface ViewerCertificateIamCertficateOptions extends ViewerCertificateBaseCertficateOptions {
+  /**
+   * Identifier of the IAM certificate.
+   */
+  readonly certificateId: string;
+}
+
+/**
+ * Viewer certificate configuration class
+ */
+export class ViewerCertificate {
+  /**
+   * Generate an AWS Certificate Manager (ACM) viewer certificate configuration
+   *
+   * @param options ACM certificate configuration options
+   * @param aliases Domain names on the certificate (both main domain name and Subject Alternative names)
+   */
+  public static acmCertificate(options: ViewerCertificateAcmCertficateOptions, ...aliases: string[]) {
```

Comment:
We usually use a "from" prefix, so it reads like a sentence `ViewerCertificate.fromAcmCertificate`

Reply:
Done